UI/src/UI.cpp

```diff
@@ -1,5 +1,9 @@
 #include "../include/ui.h"
 
```

Comment:
Can keep the include in the cpp, with a forward declaration in the header so we don't include them both in the same namespace.

Aka, main includes glad/glad.h, etc. and so does the UI header, so we would be including it twice.